### PR TITLE
docs: refactor stories into the new format vol.1

### PR DESCRIPTION
**Pull request by burakukula**
Hey there Forma 36

Just a small refactor of a couple of components to the new stories format.

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.stories.tsx [hunk 1]
```diff
@@ -18,7 +29,7 @@ const items: Item[] = [
   { value: 5, label: 'entry out of viewport' },
 ];
 
-const AutocompleteDefaultStory = ({ items }: { items: Item[] }) => {
+export const Basic = (args: AutocompleteProps<{}>) => {
```
Comment (burakukula) [inline packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.stories.tsx:32]: Can I pass the empty `{}` there? Would it be correct? 🤔

File: packages/forma-36-react-components/src/components/EntityList/EntityListItem/README.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+The EntityListItem component is used to represent lists of entities (entries and assets).
```
Comment (burakukula) [inline packages/forma-36-react-components/src/components/EntityList/EntityListItem/README.mdx:1]: I would improve those docs separately, for now, is just migration to the `README` format.

File: packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.stories.tsx (hunk 1 above)
Comment (denkristoffer) [reply packages/forma-36-react-components/src/components/Autocomplete/Autocomplete.stories.tsx:32]: You can but it's not very useful then. Maybe it's fine for the story.

**Review by denkristoffer — APPROVED** [review]
🙇

**Pull request merged**